arione-joy.md

Ainatiana opened the pull request:
Hey!
Please, have a look at this new file and feel free if it needs to be edited.
Thanks.
Ainasoa.

Ainatiana committed:
added arione-joy.md

Ainatiana committed:
Update arione-joy.md

I have edited it now so please, have a look again.

Ainatiana committed:
Update arione-joy.md

ndorfin commented on a/arione-joy.md:
```diff
@@ -0,0 +1,27 @@
+Simple. It was not until the end of 2014 that the young man signed with Mafianar. The following year, "voninkazo voarara" was broadcast on all channels and had, for weeks, been first in Madagascan hits. Arione Joy comes out of anonymity. The young singer is even approached by many producers and event organizers. Fame has not changed Arivony, which has remained the same as before. "If I want to ea ...
+Mahetsaka
+
+**Genre**: RNB,Reggae&Roots.
```
Please add spaces in the right places

ndorfin commented on a/arione-joy.md:
```diff
@@ -0,0 +1,27 @@
+
+
+
+
```
Remove these empty lines please

ndorfin reviewed (changes requested):
Please name the image filename the same as the artist page.

ndorfin commented on a/arione-joy.md:
```diff
@@ -0,0 +1,27 @@
+# Arione Joy
+
+![arione](arione-joy.jpg)
```
This is the right filename, but it'll break, because the actual image doesn't have a matching filename.

ndorfin commented on a/arione-joy.md:
```diff
@@ -0,0 +1,27 @@
+
+**Genre**: RNB,Reggae&Roots.
+
+**Artist homepage**: [facebook.com](https://web.facebook.com/pg/Arione-Joy-782590048428971/posts/)
```
Make this something like: "Arione Joy on facebook.com"

Ainatiana committed:
Update arione-joy.md

Ainatiana committed:
Update arione-joy.md

Ainatiana committed:
added arione-joy

ndorfin reviewed (changes requested):
Please remove the extra empty lines.

Ainatiana committed:
Update arione-joy.md

Hello again!
Please, let me know if my PR works now and always feel free if changes are still needed.
Thanks a lot.
Ainatiana.
